modules/dnn/test/test_layers_1d.cpp

```diff
@@ -603,4 +603,52 @@ INSTANTIATE_TEST_CASE_P(/*nothting*/, Layer_FullyConnected_Test,
                             std::vector<int>({4})
 ));
 
+typedef testing::TestWithParam<tuple<std::vector<int>>> Layer_BatchNorm_Test;
+TEST_P(Layer_BatchNorm_Test, Accuracy_01D)
+{
+    std::vector<int> input_shape = get<0>(GetParam());
+    bool has_weight = false;
+    bool has_bias = false;
```

Comment:
variables are not used besides lp.set. It means that they may be removed.

Reply:
done